listenbrainz/webserver/static/js/src/LastFMImporter.tsx

```diff
@@ -268,6 +268,12 @@ export default class LastFmImporter extends React.Component<
     this.startImport();
   };
 
+  progressBarPercentage() {
+    if (this.totalPages > this.numCompleted)
+      return (this.numCompleted / this.totalPages) * 100;
+    return 50;
```

Comment:
Could you explain this 50 to me? Is it some sort of "unknown state"?

Also, should it return 100 if this.totalPages === this.numCompleted ?

Reply:
IMO this 50 will lead to the progress bar start from half itself?